holidays/countries/myanmar.py

```diff
@@ -0,0 +1,191 @@
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import JAN, MAR, JUN, JUL, AUG, NOV, DEC
+from holidays.groups import (
+    BurmeseCalendarHolidays,
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+)
+from holidays.holiday_base import HolidayBase
+
+
+class Myanmar(
+    HolidayBase,
+    BurmeseCalendarHolidays,
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """Myanmar holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Myanmar>
+        * <https://my.wikipedia.org/wiki/မြန်မာနိုင်ငံရှိ_အားလပ်ရက်များ>
+        * [Algorithm, Program and Calculation of Myanmar Calendar](https://web.archive.org/web/20250510011425/http://cool-emerald.blogspot.com/2013/06/algorithm-program-and-calculation-of.html)
+        * [2025 International and Chinese New Year](https://web.archive.org/web/20240709104600/https://www.moi.gov.mm/news/58594)
+        * [Armed Forces Day](https://web.archive.org/web/20250522010622/https://burma.irrawaddy.com/on-this-day/2019/03/27/187691.html)
+        * [2021-2024](https://web.archive.org/web/20241112125406/https://evisa.moip.gov.mm/home/publicholiday)
+        * [2013-2025 (Webarchive)](https://web.archive.org/web/20250711015428/http://www.myanmarembassy.sg/contact-us/holidays/)
+    """
+
+    country = "MM"
+    default_language = "my"
+    # %s (estimated).
+    estimated_label = tr("%s (ခန့်မှန်း)")
+    supported_languages = ("en_US", "my")
+    # On 4 January 1948, Myanmar gained independence from British rule.
+    start_year = 1948
+
+    def __init__(self, *args, islamic_show_estimated: bool = True, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        BurmeseCalendarHolidays.__init__(self)
+        ChineseCalendarHolidays.__init__(self)
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=MyanmarIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        StaticHolidays.__init__(self, cls=MyanmarStaticHolidays)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        if self._year >= 2025:
+            # New Year's Day.
+            self._add_new_years_day(tr("နိုင်ငံတကာနှစ်သစ်ကူးနေ့"))
+
+        # Independence Day.
+        self._add_holiday_jan_4(tr("လွတ်လပ်ရေးနေ့"))
+
+        # Union Day.
+        self._add_holiday_feb_12(tr("ပြည်ထောင်စုနေ့"))
+
+        # Peasants' Day.
+        self._add_holiday_mar_2(tr("တောင်သူလယ်သမားနေ့"))
```

Comment:
According to the Myanmar version of the Wiki ( https://my.m.wikipedia.org/wiki/တောင်သူလယ်သမားနေ့ ) - the Peasants' Day first starts as January 1st in 1963, then moved to March 2nd from 1965 onwards